parser_library/src/workspaces/workspace_configuration.h

```diff
@@ -400,6 +267,8 @@ class workspace_configuration
         const shared_json& global_settings,
         external_configuration_requests* ecr);
 
+    ~workspace_configuration();
```

Comment:
I think we should delete the move constructor and assignment operator due to the reference kept in program_configuration_storage